toolchain/lc-compile/src/DOUBLE_LITERAL.t

```diff
@@ -1,6 +1,45 @@
-[0-9]+"."[0-9]+([eE][-+]?[0-9]+)? {
-  MakeDoubleLiteral(yytext, &yylval.attr[1]);
-  yysetpos();
-  return DOUBLE_LITERAL;
+	/* This contains both INTEGER_LITERAL and DOUBLE_LITERAL rules */
+	/* This ensures that they appear in this order, to ensure they match */
+	/* correctly. */
+
```

Comment:
Unfortunately not - I tried that but it makes reflex choke - it expects a .t file for each token.